app/mapper/IdMap.py

```diff
@@ -25,4 +25,5 @@ def find(object_class, objectId):
 
 # Clear the ID Map
 def clear(object_class):
-    del objectMap[object_class]
\ No newline at end of file
+    if object_class in objectMap:
+        del objectMap[object_class]
```

Comment:
@arekmano this was throwing a `KeyError` in some situations, so I added the validation first. I doubt this would cause any issues, but let me know if you think it might.